mappings/net/minecraft/state/property/Properties.mapping

```diff
@@ -2,23 +2,23 @@ CLASS bwb net/minecraft/state/property/Properties
 	FIELD A TRIGGERED Lbwc;
 	FIELD B UNSTABLE Lbwc;
 	FIELD C WATERLOGGED Lbwc;
-	FIELD D AXIS_XZ Lbwi;
-	FIELD E AXIS_XYZ Lbwi;
-	FIELD F UP_BOOL Lbwc;
-	FIELD G DOWN_BOOL Lbwc;
-	FIELD H NORTH_BOOL Lbwc;
-	FIELD I EAST_BOOL Lbwc;
-	FIELD J SOUTH_BOOL Lbwc;
-	FIELD K WEST_BOOL Lbwc;
+	FIELD D HORIZONTAL_AXIS Lbwi;
```

Comment:
Plural of `AXIS` is `AXES`, so `HORIZONTAL_AXES`. Same below.

Reply:
It represents a key for a singular value: `state.get(AXIS)`. If you argue that this should be plural you should argue that FACING should be FACINGS too. But it is incorrect naming for such a property imo.

Reply:
nice new name! old name is confusing as i thought it was a swizzle. please mark this as resolved.